webauthn-rs-core/Cargo.toml

```diff
@@ -33,6 +33,7 @@ rand = { version = "0.8" }
 url = { version = "2", features = ["serde"] }
 x509-parser = "0.13.0"
 der-parser = "7.0.0"
+jwt-simple = "0.11.0"
```

Comment:
Can't use jwt-simple as it's not an openssl backed crypto library (and it has other issues .... turns out every jwt library in rust is fucked in some way )

We'll probably need to swap to https://github.com/kanidm/compact-jwt which we maintain.

Reply:
I have a draft right now to make compact jwt work with safety net. The main thing is we need support for x5c pubkeys.

Reply:
We could merge this as is-and then rework to have compact jwt?